cmd/ocsp-responder/main.go

```diff
@@ -70,31 +69,41 @@ serialNumber field, since we will always query on it.
 
 */
 type DBSource struct {
-	db        *sql.DB
+	dbMap     *gorp.DbMap
 	caKeyHash []byte
 }
 
 func NewSourceFromDatabase(dbMap *gorp.DbMap, caKeyHash []byte) (src *DBSource, err error) {
-	src = &DBSource{db: dbMap.Db, caKeyHash: caKeyHash}
+	src = &DBSource{dbMap: dbMap, caKeyHash: caKeyHash}
 	return
 }
 
 const responseQuery = "SELECT resposne FROM ocsp_responses WHERE serialNumber"
 
 func (src *DBSource) Response(req *ocsp.Request) (response []byte, present bool) {
+	log := blog.GetAuditLogger()
+
 	// Check that this request is for the proper CA
 	if bytes.Compare(req.IssuerKeyHash, src.caKeyHash) != 0 {
+		log.Debug(fmt.Sprintf("Request intended for CA Cert ID: %s", hex.EncodeToString(req.IssuerKeyHash)))
 		present = false
 		return
 	}
 
-	// SELECT resposne FROM ocsp_responses WHERE serialNumber
-	err := src.db.QueryRow(responseQuery).Scan(&response)
+	serialString := fmt.Sprintf("%032x", req.SerialNumber)
```

Comment:
ha! Heck, I recommended that the creation of that function in the first place. I guess it's Friday.